modules/administration/pages/disconnected-setup.adoc

```diff
@@ -151,75 +155,50 @@ SMT:: https://documentation.suse.com/sles/12-SP5/single-html/SLES-smt/index.html
 
 
 
-== Synchronize a Disconnected Server
 
-When you have removable media loaded with your {scc} data, you can use it to synchronize your disconnected server.
+[[disconnected-setup-server]]
+== Deploy and Synchronize a Disconnected Server
 
+To set up {productname} as a disconnected server follow the instructions of an air-gapped deployment.
+It is recommended to deploy it as a Virtual Machine (VM) using a provided image.
+For an air-gapped deployment of {productname} Server, see xref:installation-and-upgrade:container-deployment/suma/server-air-gapped-deployment-suma.adoc[].
 
-//// 
-OLD
-.Procedure: Synchronizing a Disconnected Server
-. Mount your removable media device to the {productname} server.
-    In this example, the mount point is [path]``/media/disk``.
-. Open ``/etc/rhn/rhn.conf`` and define the mount point by adding or editing this line:
-+
-----
-server.susemanager.fromdir = /media/disk
-----
-+
-. Restart the Tomcat service:
-+
-----
-systemctl restart tomcat
-----
-. Refresh the local data:
-+
-----
-mgr-sync refresh
-----
-. Perform a synchronization:
+**IMPORTANT:** Execute the final command with the [option]``--mirrorPath`` option (replacing [literal]``</media/disk>`` with your mount point):
```

Comment:
```suggestion
[IMPORTANT
====
 Execute the final command with the [option]``--mirrorPath`` option (replacing [literal]``</media/disk>`` with your mount point).
====
```

Reply:
Ok, but I solved it differently with partial re-writing.